oc/openshift_ops.py

```diff
@@ -184,3 +185,165 @@ def create_project(self, project):
             raise Exception(err)
 
         return _rc
+
+    def create_cephblockpool(
+        self,
+        namespace,
+        cephblockpool_name_prefix="autotests-cephblockpool",
+        **kwargs
+    ):
+        """
+        Creates cephblock pool
+
+        Args:
+            namespace (str): namespace to create cephblockpool
+            cephblockpool_name_prefix (str): prefix given to cephblockpool
+
+        kwargs:
+            The keys, values in kwargs are:
+                - failureDomain : (str)
+                - size : (int)
+
+        Returns:
+            str : name of the cephblockpool created
+
+        Raises:
+            KeyError when error occured
+
+        Examples:
+            create_cephblockpool(
+                namespace,
+                cephblockpool_name_prefix="autotests-blockpool",
+                failureDomain="host",
+                size=3)
+
+        """
+        service_cbp = self.dyn_client.resources.get(
+            api_version='v1',
+            kind='CephBlockPool'
+        )
+        cephblockpool_name = "%s-%s" % (
+            cephblockpool_name_prefix,
+            get_random_str()
+        )
+        # overwrite the namespace with openshift-storage, since cephblockpool
+        # is tied-up with openshift-storage
+        namespace = "openshift-storage"
+        cephblockpool_data = {}
+        cephblockpool_data['cephblockpool_name'] = cephblockpool_name
+        cephblockpool_data['rook_api_version'] = defaults.rook_api_version
+        cephblockpool_data['failureDomain'] = kwargs.get('failureDomain')
+        cephblockpool_data['size'] = kwargs.get('size')
+
+        data = generate_data("cephblockpool.yaml", cephblockpool_data)
```

Comment:
See my other comment in utils.py about the yaml file.
I think we should pass here the yaml file without the need to manipulating it in the generate_data function cause in case I'm creating a local yaml in this file, it will not work

Reply:
I removed the this function since we have another function ( generate_yaml_from_jinja2_template_with_data ) which was merged recently which served the same purpose. generate_yaml_from_jinja2_template_with_data